csharp/extractor/Semmle.Extraction.CSharp/Entities/File.cs

```diff
@@ -61,7 +61,7 @@ public override void Populate(TextWriter trapFile)
                 }
             }
 
-            trapFile.file_extraction_mode(this, Context.Extractor.Standalone ? 1 : 0);
+            trapFile.file_extraction_mode(this, Context.Extractor.Mode);
```

Comment:
Now printing the "entire" mode and not only, whether the it is standalone extraction to the trap file. This is this intentional?

Reply:
Yes, we want to persist all bit fields.